src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -843,13 +845,44 @@ public void testPrivateKeyInConnectionString() throws SQLException, IOException
           (int) ErrorCode.INVALID_OR_UNSUPPORTED_PRIVATE_KEY.getMessageCode(), e.getErrorCode());
     }
     connection.close();
+  }
 
-    // clean up
-    connection = getConnection();
-    statement = connection.createStatement();
-    statement.execute("use role accountadmin");
-    statement.execute(String.format("alter user %s unset rsa_public_key", testUser));
-    connection.close();
+  @Test
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
+  public void testPrivateKeyInConnectionStringWithBouncyCastle() throws SQLException, IOException {
+    System.setProperty(ENABLE_BOUNCYCASTLE_PROVIDER_JVM, "true");
+    testPrivateKeyInConnectionString();
+  }
+
+  @Test
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
+  public void testPbes2Support() throws SQLException, IOException {
+    // We're testing if we can decrypt a private key generated using OpenSSL v3.
+    // This was failing before with an invalid private key error before
+    // SNOW-896618 added the ability to decrypt private keys with
+    // -Dnet.snowflake.client.bouncyCastle=true
```

Comment:
the parameter is different - `net.snowflake.jdbc.enableBouncyCastle` - let's make this comment a part of javadoc of the test method and refer to the constant in it